core/zoe/docs/SWAPS.md

```diff
@@ -0,0 +1,78 @@
+# Swaps
+
+If I want to trade one kind of asset for another kind, I could send
+you the asset and ask you to send me the other kind back. But, you
+could choose to behave opportunistically: receive my asset and give
+nothing back. To solve this problem, swap contracts allow users to
+securely trade one kind of eright for another kind, leveraging Zoe for
+escrow and offer-safety. At no time does any user have the ability to
+behave opportunistically.
+
+## Simple Offer Swap
+
+In the "Simple Offer" swap, the governing contract is the
+`simpleOffer` interface with the `swapSrcs` installed. 
+
+Let's say that Alice wants to create a swap that anyone can be the
+counter-party for. She creates the zoeInstance:
+
+```js
+const makeSimpleSwap = makeSimpleOfferMaker(swapSrcs);
+const { zoeInstance, governingContract: simpleSwap } = zoe.makeInstance(
+  makeSimpleSwap,
+  issuers,
+);
+```
+
+Then escrows her offer with the zoeInstance and gets an escrowReceipt
+and a claimPayoff ERTP payment from which she can get her winnings:
+
+```js
+const aliceOfferDesc = harden([
+  {
+    rule: 'offerExactly',
+    amount: moolaIssuer.makeAmount(3),
+  },
+  {
+    rule: 'wantExactly',
+    amount: simoleanIssuer.makeAmount(7),
+  },
+]);
+const alicePayments = [aliceMoolaPayment, aliceSimoleanPayment];
```

Comment:
Good point. Alice doesn't need to send in a simoleans payment object. She can send in `undefined` for that element in the array. If she passes in a payment, it will be ignored. I should remedy this in the tests and documents